app/views/start.html

```diff
@@ -53,33 +53,21 @@ <h2 class="govuk-heading-m">Before you start</h2>
         {% elseif data.features['npq-accounts'].on %}
 
       {% else %}
-      <p>You will be able to use this service to register:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>to study for an NPQ, including NPQ for Early Years Leadership</li>
-        <li>for the Early Headship Coaching Offer</li>
-      </ul>
-
-      <h2 class="govuk-heading-m">Frequently asked questions</h2>
-      <p>Read our FAQs if you:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>work in <a href="/faq/early-years">early years or childcare</a></li>
-        <li>work in a <a href="/faq/schools">school, academy trust or 16 to 19 educational setting</a></li>
-        <li><a href="/faq/other-users">do not work in a school, childcare or early years</a>, for example you work in a virtual school, hospital school or young offender institution</li>
-    
-      </ul>
+      <p>Use this service to register for an NPQ or the early headship coaching offer.</p>
+      <p>You also need to apply separately with your training provider. They’ll contact you once you’ve registered.</p>
 
+     
         <h2 class="govuk-heading-m">Before you start</h2>
-        <p>You must:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>have a teacher reference number (TRN)</li>
-          <li>have your funding source confirmed – either a scholarship, school, college or personal funding</li>
-          <li>know which NPQ course you want to study</li>
-          <li>have chosen a provider</li>
-        </ul>
+        
+        <p><a href="/register/choosing-an-npq">Choose your NPQ and provider</a>.</p>
```

Comment:
```suggestion
        <p><a href="/register/choosing-an-npq">Choose an NPQ and provider</a>.</p>
```

Reply:
For consistency with new heading: https://github.com/DFE-Digital/npq-prototype/pull/38